MeretzLib/Apple/iOS/Meretz/Meretz/Meretz.m

```diff
@@ -685,4 +761,91 @@ - (MeretzTask *) getTask: (MeretzTaskId) taskId
 		return result;
 	}
 
+	// calls appropriate delegate method to signal task completion
```

Comment:
This new method allows the Meretz API to call the appropriate delegate (if available) upon task completion.